### DISCUSSION TITLE
video_params unknown config option

### DISCUSSION THREAD

**Discussion by peakpeak-github** [Motion-Project/motion#1714] [Q&A] [answered]
motion Version 4.3.2 running on Raspberry Pi 4
The program is showing the video stream ok. However, I'd like to adjust the white balance and other parameters.
According to the manual: 

_video_params
Type: String
Range / Valid values: Max 4095 characters
Default: Not defined
For most devices the configuration options are automatically assigned but devices also usually allow the user to change certain settings to suit their needs. The options that the camera manufacturer permits the user to change are called "controls". The [video_params](https://motion-project.github.io/motion_config.html#video_params) option allows users to specify values for these controls when Motion runs._

In /etc/motion/motion.conf I enter:
video_params White Balance, Automatic

When starting motion I get this:
_pi@raspen:~ $ sudo motion
[0:motion] [NTC] [ALL] conf_load: Processing thread 0 - config file /etc/motion/motion.conf
[0:motion] [ALR] [ALL] **conf_cmdparse: Unknown config option "video_params"**
[0:motion] [NTC] [ALL] motion_startup: Logging to file (/var/log/motion/motion.log)_

**Comment by tosiara** [comment]
Use latest version 4.5.x

**Reply by peakpeak-github** [reply, marked as answer]
With some effort to grab the latest version it works!
